src/client/components/loginComponent/index.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input, Button, Form, Header, Grid, Segment, Message } from 'semantic-ui-react';
+
+export class LoginComponent extends React.Component {
+
+    handleChange = (item) => {
+        this.props.change(item.target);
```

Comment:
This means, that when some component uses your component, it will have those props that you defined. Imagine this:
```javascript
<LoginComponent click={} change={} />
```
What doesn't it mean: login component was clicked/changed? How to handle that?